docs/content/guides/dagster/managing-ml.mdx

```diff
@@ -0,0 +1,271 @@
+
+```python file=/guides/dagster/managing_ml/managing_ml_code.py startafter=without_policy_start endbefore=without_policy_end
+from dagster import AutoMaterializePolicy, FreshnessPolicy, asset
+
+
+@asset
+def some_data():
+    ...
+
+
+@asset(auto_materialize_policy=AutoMaterializePolicy.lazy())
+def some_ml_model(some_data):
+    ...
+
+
+@asset(
+    auto_materialize_policy=AutoMaterializePolicy.lazy(),
+    freshness_policy=FreshnessPolicy(maximum_lag_minutes=7 * 24 * 60),
+)
+def predictions(some_ml_model):
+    ...
+```
+
+We can also use a more traditional schedule on our machine learning assets, causing them to be re-materialized or retrained on the latest data. For example, setting up a [cron schedule on a daily basis](/concepts/partitions-schedules-sensors/schedules).
+
+This can be useful if you have data that is also being scheduled on a cron schedule and want to add your machine model jobs to run on a schedule as well.
+
+```python file=/guides/dagster/managing_ml/managing_ml_code.py startafter=basic_schedule_start endbefore=basic_schedule_end
+from dagster import AssetSelection, define_asset_job, ScheduleDefinition
+
+ml_asset_job = define_asset_job("ml_asset_job", AssetSelection.groups("ml_asset_group"))
+
+basic_schedule = ScheduleDefinition(job=ml_asset_job, cron_schedule="0 9 * * *")
+```
+
+### Monitoring
+
+Integrating your machine learning models into Dagster allows you to see when the model and its data dependencies were refreshed, or when a refresh process has failed. By using Dagster to monitor performance changes and process failures on your ML model, it becomes possible to set up remediation paths, such as automated model retraining, that can help resolve issues like model drift.
+
+In this example, the model is being evaluated against the previous model’s accuracy. If the model’s accuracy has improved, the model is returned for use in downstream steps, such as inference or deploying to production.
+
+```python file=/guides/dagster/managing_ml/managing_ml_code.py startafter=conditional_monitoring_start  endbefore=conditional_monitoring_end
+from dagster import RunRequest, asset, Output
+from sklearn import linear_model
+
+
+@asset
+def other_ml_model(training_data, test_data, model_accuracy):
+    reg = linear_model.LinearRegression()
+    reg.fit(training_data)
+    new_model_accuracy = reg.score(test_data)
+    if new_model_accuracy > model_accuracy:
+        yield Output(reg, metadata={"model_accuracy": new_model_accuracy})
+```
+
+We can also monitor the score or other information of a model through alerts, such as email or Slack. In the example below, we set up a Slack message to post the score of `ml_model` upon success. The message can be customized to include information such as which features were most influential in model training, the volume of data that the model was trained on, or in this case, the score of the model.
```

Comment:
I don't see anything related to slack in the example below.

Reply:
had the wrong code referenced